src/Model/addNumbers.js

```diff
@@ -0,0 +1,14 @@
+const addNumbers = userInput => {
+  const REGULAR_EXPRESSION = /[^0-9]/g;
+  const REMOVE_STR = userInput.replace(REGULAR_EXPRESSION, '');
+
+  const SPLIT_NUMBERS = REMOVE_STR.split('');
+  let totalNumberSum = 0;
+
+  for (let i = 0; i < SPLIT_NUMBERS.length; i += 1) {
+    totalNumberSum += parseInt(SPLIT_NUMBERS[i], 10);
+  }
+  return totalNumberSum;
+};
+
```

Comment:
이 부분에선 for문으로 해도 되지만 reduce로 하는 것도 괜찮은 방법일 것 같아요 

```suggestion
return REMOVE_STR.split('').reduce((sum, num) => sum + parseInt(num, 10), 0);
```